index.html

```diff
@@ -100,11 +100,21 @@
 <pre><code>
   events.on('route:/profile:get', function (connection) {
     connection.res.end('route /profile now responding to get requests');
+    // If those more familiar with using ".send()" when finishing
+    // eventhandlers it is also available as an alternative,
+    // like this:
+    // connection.res.send('route /profile now responding to get requests');
```

Comment:
probably want to prefer `.send` over `.end` in the docs. It provides a better developer experience and less work when implementing sites.